app/services/payment_providers/stripe/webhooks/payment_intent_succeeded_service.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module PaymentProviders
+  module Stripe
+    module Webhooks
+      class PaymentIntentSucceededService < BaseService
```

Comment:
:+1: It seems it was in your mind as well :smile:

Reply:
Yes! I went back and forth with the dedicated service. Sorry, I didn't realise the PR wasn't draft 🙏